deno/payloads/v10/channel.ts

```diff
@@ -196,7 +201,7 @@ export interface APIGroupDMChannel extends Omit<APIDMChannelBase<ChannelType.Gro
 }
 
 export interface APIThreadChannel
-	extends APIGuildChannel<ChannelType.PublicThread | ChannelType.PrivateThread | ChannelType.AnnouncementThread> {
+	extends APITextBasedChannel<ChannelType.PublicThread | ChannelType.PrivateThread | ChannelType.AnnouncementThread> {
```

Comment:
This feels wrong, thread channels are guild ones

Reply:
Whoops you're right, I overlooked the type regression there from guild to basic 😅 - I'll make it extend both, as all the properties in `APIGuildTextChannel` are not applicable to threads.